configs/data/im2im/diffae.yaml

```diff
@@ -0,0 +1,136 @@
+_target_: cyto_dl.datamodules.dataframe.DataframeDatamodule
+
+path:
+cache_dir:
+
+num_workers: 0
+batch_size: 1
+pin_memory: True
+split_column:
+columns:
+  - ${source_col}
+
+transforms:
+  train:
+    _target_: monai.transforms.Compose
+    transforms:
+      # channels are [blank, membrane,blank, structure, blank, nuclear dye, brightfield ]
+      - _target_: monai.transforms.LoadImaged
+        keys: ${source_col}
+        reader:
+          - _target_: cyto_dl.image.io.MonaiBioReader
+            # NOTE: eval is used so only the experiment file is required to change for beginning users. This is not recommended when creating your own configs.
+            dimension_order_out: ${eval:'"CZYX" if ${spatial_dims}==3 else "CYX"'}
+            C: 5
+            Z: ${eval:'None if ${spatial_dims}==3 else 38'}
+      - _target_: monai.transforms.Zoomd
+        keys: ${source_col}
+        zoom: 0.25
+        keep_size: False
+      - _target_: monai.transforms.ToTensord
+        keys: ${source_col}
+      - _target_: monai.transforms.ScaleIntensityRangePercentilesd
+        keys: raw
+        lower: 1
+        upper: 99
+        b_min: 0
+        b_max: 1
+        clip: True
+      - _target_: monai.transforms.RandSpatialCropSamplesd
+        keys:
+          - ${source_col}
+        roi_size: ${data._aux.patch_shape}
+        num_samples: 1
+        random_size: False
+
+  test:
+    _target_: monai.transforms.Compose
+    transforms:
+      # channels are [blank, membrane,blank, structure, blank, nuclear dye, brightfield ]
```

Comment:
what are blanks?

Reply:
just channels in variance data without a structure imaged